_data/simple-icons.json

```diff
@@ -7950,6 +7950,11 @@
             "hex": "047AED",
             "source": "https://www.keycdn.com/logos"
         },
+        {
+            "title": "Keycloak",
+            "hex": "4D4D4D",
```

Comment:
They use `#428BCA` for `btn-primary`  and `--bs-primary-rgb` in [keycloak.css](https://www.keycloak.org/resources/css/keycloak.css)... But I'm not sure it is really representative of the brand...

Reply:
Can't figure out what the brand colour is supposed to be at all but, if we're going with a blue, would we be better off using one of the blues from the logo?